mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/GEOR_custom.js

```diff
@@ -0,0 +1,80 @@
+Ext.namespace("GEOR");
+
+GEOR.custom = {
```

Comment:
You're absolutely right that there is code duplication here.
That's why I was talking about "technical debt" yesterday: maybe we should move all the config options from GEOR_custom.js to a server-side properties file, and have a controller dedicated to passing these config options from the server to the client-side in both cases (I mean with and without the data dir activated). Maybe we could also insert these options into the ModelAndView that mapfishapp's HomeController handles.